internal/sdkprovider/service/kafkaschema/kafka_schema.go

```diff
@@ -79,13 +105,40 @@ func diffSuppressJSONObject(_, old, new string, _ *schema.ResourceData) bool {
 	return reflect.DeepEqual(objNew, objOld)
 }
 
+// diffSuppressJSONObjectOrProtobufString checks logical equivalences in JSON or Protobuf Kafka Schema values.
+func diffSuppressJSONObjectOrProtobufString(k, old, new string, d *schema.ResourceData) bool {
+	if ok := protobufHeaderRegExp.Match([]byte(old)); ok {
+		return normalizeProtobufString(old) == normalizeProtobufString(new)
+	}
+
+	return diffSuppressJSONObject(k, old, new, d)
+}
+
 // normalizeJSONString returns normalized JSON string
 func normalizeJSONString(v interface{}) string {
 	jsonString, _ := structure.NormalizeJsonString(v)
 
 	return jsonString
 }
 
+// normalizeProtobufString returns normalized Protobuf string.
+func normalizeProtobufString(i any) string {
```

Comment:
Same point here, I would love to do normalization using a go protobuf library for that.

Reply:
it requires the end users to install `protoc`, which wouldn't be too good